SPECS/python-tensorflow-io-gcs/python-tensorflow-io-gcs.spec

```diff
@@ -0,0 +1,50 @@
+%global pypi_name tensorflow-io-gcs
+%global _description %{expand:
+TensorFlow I/O is a collection of file systems and file formats that are not available in TensorFlow's built-in support}
+%define _enable_debug_package 0
+%global debug_package %{nil}
+
+Summary:        TensorFlow I/O is a collection of file systems and file formats that are not available in TensorFlow's built-in support
+Name:           python-%{pypi_name}
+Version:        0.29.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/tensorflow/io
+Source0:        https://github.com/tensorflow/io/archive/refs/tags/v%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+BuildRequires:  python3-six
+ExclusiveArch:  x86_64
```

Comment:
The package seems to contain only Python scripts. Why is it exclusively aimed at AMD64?